core/src/main/scala/kafka/server/BrokerFeatures.scala

```diff
@@ -93,6 +93,9 @@ object BrokerFeatures extends Logging {
               feature.latestProduction
             }))
     }
+    if (unstableFeatureVersionsEnabled) {
+      features.put("kraft.version", new SupportedVersionRange(1))
```

Comment:
I will do that in a follow-on PR.

Reply:
I'm a little confused here. This has one argument? And is kraft.version part of this PR?

Reply:
> I'm a little confused here. This has one argument?

There's a constructor that creates a range 0-N based on just being given N. I will change it to be the two-argument constructor since I guess that's confusing.

> And is kraft.version part of this PR?

I want to have it here for test purposes. I will create it for real in a follow-on PR, if this PR ever concludes.

Reply:
I guess I was confused because I didn't see it in the file. https://github.com/apache/kafka/blob/0b11971f2c94f7aadc3fab2c51d94642065a72e5/clients/src/main/java/org/apache/kafka/clients/admin/SupportedVersionRange.java#L38 but 🤷‍♀️